components/remote-environment-broker/internal/broker/provision.go

```diff
@@ -220,10 +221,15 @@ func (svc *ProvisionService) createEaOnSuccessProvision(reName, reID, ns string,
 				SourceID:    reName,
 			},
 		})
-	if err != nil {
+	switch {
+	case err == nil:
+		svc.log.Infof("Created EventActivation: [%s], in namespace: [%s]", reID, ns)
+	case apiErrors.IsAlreadyExists(err):
+		svc.log.Infof("%v: EventActivation `%s` already exists", err, reID)
```

Comment:
you should perform update action in case of "already exists" error. In this scenario, you are not updating the owner reference which was changed. Please also add a business comment why we are performing such action. Describe a scenario when sth like this can happened because this is not a normal happy-path.

Reply:
fixed and added comment